library/pod-security-policy/proc-mount/template.yaml

```diff
@@ -16,6 +16,10 @@ spec:
           properties:
             procMount:
               type: string
+              description: "Defines the strategy for the security exposure of certain paths in `/proc` by the container runtime."
+              enum:
+                - Default
```

Comment:
It'd be cool if we could document the significance of specific enum values.

Reply:
So `Enum` [is a slice](https://github.com/kubernetes/apiextensions-apiserver/blob/master/pkg/apis/apiextensions/types_jsonschema.go) of a `JSON` [objects](https://github.com/kubernetes/apiextensions-apiserver/blob/0149c93f156a9bb358230f5fbdca12516c3e42d4/pkg/apis/apiextensions/types_jsonschema.go#L129).  That is actually an `interface{}` type, but I think that's just b/c it has to correspond to whatever is declared in `type:`.

So, I think our only option would be to write details about the enum values in the `description` on line 19.  Proposed version:

```Defines the strategy for the security exposure of certain paths in `/proc` by the container runtime. Setting to `Default` will use the runtime defaults, where `Unmasked` bypasses the default behavior.```

Reply:
Yeah, I think you're right. Pitty, the ability to have flexible formatting options around enum docstrings could be useful.

Reply:
updated with my proposed description